charts/dbildungs-iam-keycloak/dev-realm-spsh.json

```diff
@@ -454,6 +455,24 @@
     "webAuthnPolicyPasswordlessAvoidSameAuthenticatorRegister": false,
     "webAuthnPolicyPasswordlessAcceptableAaguids": [],
     "users": [
+      {
+        "id": "7baf74aa-565f-4cfc-9d5a-8f1a3f374dc9",
+        "createdTimestamp": 1727357679883,
+        "username": "service-account-spsh-service",
+        "enabled": true,
+        "totp": false,
+        "emailVerified": false,
+        "serviceAccountClientId": "spsh-service",
+        "disableableCredentialTypes": [],
+        "requiredActions": [],
+        "clientRoles": {
+          "realm-management": [
+            "manage-realm"
```

Comment:
@tikle-cap @kristoff-kiefer We need "manage-clients" here. Sorry for the confusion.

Reply:
replaced